packages/odyssey-storybook/src/index.ts

```diff
@@ -16,6 +16,9 @@ import type { PropItem } from "react-docgen-typescript";
 import sass from "sass";
 import postcss from "postcss";
 
+const isProduction = process.env.NODE_ENV === "production";
+const withStyles = /\/(Banner|Button)\.module\.scss$/;
```

Comment:
the idea is to continue to add to this regex as we update more components. For now, because our tooling does not support hot module reload (but `style-loader` does) we'll use the existing webpack pipeline but production storybook builds will use our HOC